src/uikit/Select.js

```diff
@@ -3,91 +3,179 @@ import Downshift from 'downshift';
 
 import downChevronIcon from '../assets/icon-chevron-down-grey.svg';
 
+export const OptionDropdownWrapperCss = `
+  position: absolute;
+  background: white;
+  min-width: 100%;
+  z-index: 1;
+  border: 1px solid rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+  cursor: pointer;
+  padding: 5px;
+  top: 100%;
+`;
+
+const DisabledDropdownOption = `
```

Comment:
same not here re: css patterns, but again maybe that's in the refactor ticket... same note re: variable name tho